jackson2/src/main/java/org/jdbi/v3/jackson2/Jackson2Config.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jdbi.v3.jackson2;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import org.jdbi.v3.core.config.JdbiConfig;
+import org.jdbi.v3.meta.Beta;
+
+/**
+ * Configuration class for Jackson 2 integration.
+ */
+@Beta
+public class Jackson2Config implements JdbiConfig<Jackson2Config> {
+    private ObjectMapper mapper;
```

Comment:
Do we want to default this to a `new ObjectMapper()` for basic out of the box functionality?

Reply:
I waffled a bit on that one.  Personally, I always configure my Mappers, so this prevents me from accidentally forgetting to configure it.  It also means whatever mapper caching Jackson does is shared with the rest of the application.  I'm happy to change this if anyone feels strongly, though.

Reply:
I usually configure the module 2, but for those who don't need to I'd rather have it "just work" out fo the box.

An alternative to having a default is to no-arg constructor to the plugin that uses a `new ObjectMapper()`

Reply:
I think it feels more in line with how the rest of jdbi works if we default this to a `new`